### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by KevinBurton** [IvanJosipovic/BlazorTable#268]
The installation instructions state that on should install the package BlazorTable. But for a client side Blazor application there are two projects, one for the client and one for the server. It is not clear which one (or both) the package should be installed to. Also the last instruction states that the Middleware should be installed in Program.cs or Startup.cs. It is not clear how to decide which.

**Comment by conficient** [comment]
Agreed it might be confusing if someone used the ASP.NET hosted WASM project as to where the changes should be made.
